package.json

```diff
@@ -25,8 +25,7 @@
     "clean": "rm -rf packages/*/tsconfig.tsbuildinfo && rm -rf coverage test-report && lerna clean --yes && rm -rf node_modules && yarn cache clean",
```

Comment:
and really this should probably look like
```suggestion
    "clean": "lerna run clean --scope @shapeshiftoss/* && rm -rf coverage test-report node_modules && yarn cache clean",
```

Reply:
I don't think I changed that line but I can add it.

Reply:
I just noticed it when looking at the build script. Thanks